clustered/server/src/main/java/org/ehcache/clustered/server/ServerStoreImpl.java

```diff
@@ -90,4 +90,30 @@ public void clear() {
   public void close() {
     store.close();
   }
+
+  // stats
+
+  public long allocatedMemory() {return store.allocatedMemory();}
```

Comment:
@ljacomet : I think the answer is yes because we could make `ServerStoreImpl` implements `MapInternals`, and then register stats over any `MapInternals` object.